io.cpp

```diff
@@ -18,7 +18,7 @@ int dirExists(const string& pathName)
 
     if( stat( pathName.c_str(), &info ) != 0 )
         return -1; // Cannot acces path
-    else if( info.st_mode & S_IFDIR )  // S_ISDIR() doesn't exist on my windows 
+    else if( info.st_mode & S_IFDIR )  // S_ISDIR() doesn't exist on my windows
```

Comment:
I noticed that `io.cpp` uses LF line terminators, while `hemoflow.cpp` uses CRLF. I am not sure if this is intentional. We can discuss it on Thursday.